f1/index.d.ts

```diff
@@ -25,9 +25,9 @@ interface F1 extends NodeJS.EventEmitter {
 	init(initState: string): F1;
 
 	destroy(): void;
-	apply(): void;
+	apply(pathToTarget: string, target: any, parserDefinition: any): void;
 
-	go(state: string, callback?: (...args: any[]) => void): F1;
+	go(state: string, callback?: () => void): F1;
```

Comment:
I think neither is correct actually.. looking at https://github.com/mikkoh/kimi/blob/master/index.js#L143, seems like you get two parameters. let me fix this.